tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
+		},
+		{
+			name:              "REACHABLE_wikipedia.org_over_cidr_range",
+			config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
+			url:               "https://www.wikipedia.org",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_google.com_over_cidr_range",
+			config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
+			url:               "https://google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_in.google.com_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google.yaml",
+			url:               "https://www.google.co.in/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "REACHABLE_uk.google.com_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google.yaml",
+			url:               "https://www.google.co.uk/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_www.bing.org_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google.yaml",
+			url:               "https://www.bing.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_uk.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.co.uk/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "REACHABLE_in.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.co.in/",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "https://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_uk.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.co.uk/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "UNREACHABLE_in.google.com_no_vs_over_google_wildcard_80",
+			config:            "testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml",
+			url:               "http://www.google.co.in/",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_in.google.com_no_vs_over_google_wildcard",
+			config:            "testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml",
+			url:               "https://www.google.co.in/",
```

Comment:
skip all the above as the yahoo test will test that you can reach domains without a virtual service